client/me/profile-links-add-wordpress/index.jsx

```diff
@@ -60,46 +57,49 @@ export default React.createClass( {
 		for ( inputName in this.state ) {
 			if ( 'site-' === inputName.substr( 0, 5 ) && this.state[ inputName ] ) {
 				siteID = parseInt( inputName.substr( 5 ), 10 ); // strip leading "site-" from inputName to get siteID
-				site = sites.getSite( siteID );
-				links.push( {
-					title: site.name,
-					value: site.URL
-				} );
+				site = sites[ siteID ];
+				if ( site ) {
+					links.push( {
+						title: site.name,
+						value: site.URL
+					} );
+				}
```

Comment:
Yeah, I agree totally. It's a very obsolete pattern that we should consider refactoring in a future PR.